core/capabilities/triggers/logevent/service.go

```diff
@@ -0,0 +1,154 @@
+package logevent
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+)
+
+const ID = "log-event-trigger-%s-%d@1.0.0"
+
+const defaultSendChannelBufferSize = 1000
+
+var logEventTriggerInfo = capabilities.MustNewCapabilityInfo(
+	ID,
+	capabilities.CapabilityTypeTrigger,
+	"A trigger that listens for specific contract log events and starts a workflow run.",
+)
+
+// Log Event Trigger Capability Input
+type Input struct {
+}
+
+// Log Event Trigger Capability Payload
+type Payload struct {
+	// Time that Log Event Trigger's task execution occurred (RFC3339Nano formatted)
+	ActualExecutionTime string
+}
+
+// Log Event Trigger Capability Response
+type Response struct {
+	capabilities.TriggerEvent
+	Metadata struct{}
+	Payload  Payload
+}
+
+// Log Event Trigger Capabilities Manager
+// Manages different log event triggers using an underlying triggerStore
+type LogEventTriggerService struct {
+	capabilities.CapabilityInfo
+	capabilities.Validator[RequestConfig, Input, capabilities.TriggerResponse]
+	lggr           logger.Logger
+	triggers       CapabilitiesStore[logEventTrigger, capabilities.TriggerResponse]
+	relayer        core.Relayer
+	logEventConfig LogEventConfig
+}
+
+// Common capability level config across all workflows
+type LogEventConfig struct {
+	ChainId        uint64 `json:"chainId"`
+	Network        string `json:"network"`
+	LookbackBlocks uint64 `json:"lookbakBlocks"`
+	PollPeriod     uint64 `json:"pollPeriod"`
+}
+
+type Params struct {
+	Logger         logger.Logger
+	Relayer        core.Relayer
+	RelayerSet     core.RelayerSet
+	LogEventConfig LogEventConfig
+}
+
+var _ capabilities.TriggerCapability = (*LogEventTriggerService)(nil)
+var _ services.Service = &LogEventTriggerService{}
+
+// Creates a new Cron Trigger Service.
+// Scheduling will commence on calling .Start()
+func NewLogEventTriggerService(p Params) *LogEventTriggerService {
+	l := logger.Named(p.Logger, "LogEventTriggerCapabilityService: ")
+
+	logEventStore := NewCapabilitiesStore[logEventTrigger, capabilities.TriggerResponse]()
+
+	return &LogEventTriggerService{
+		CapabilityInfo: logEventTriggerInfo,
```

Comment:
I think you will need a new Validator in here.

I used your PR as a template for parsing the config, but it failed without
```
		Validator:           capabilities.NewValidator[TriggerConfig, Input, capabilities.TriggerResponse](capabilities.ValidatorArgs{Info: webapiTriggerInfo}),
```

Reply:
Added this. Thanks!